Gems/EMotionFX/Code/EMotionFX/Tools/EMotionStudio/Plugins/StandardPlugins/Source/MotionWindow/MotionExtractionWindow.cpp

```diff
@@ -355,6 +428,49 @@ namespace EMStudio
         }
     }
 
+    void MotionExtractionWindow::OnRootMotionCheckboxClicked()
+    {
+        const CommandSystem::SelectionList& selectionList = GetCommandManager()->GetCurrentSelection();
+        const size_t numSelectedMotions = selectionList.GetNumSelectedMotions();
+
+        if (numSelectedMotions == 1)
+        {
+            EMotionFX::Motion* curMotion = selectionList.GetMotion(0);
+            auto rootMotionDataPtr = curMotion->GetRootMotionExtractionData();
```

Comment:
uber nit: `auto*` for clarity about the type here

Reply:
it's also smart ptr. Maybe calling it  xxxPtr suggest it would be raw ptr? is that more common?